main.go

```diff
@@ -101,9 +101,6 @@ func showTopLevel(tmpl *template.Template, src *retrodep.GoSource) *retrodep.Ref
 		display(tmpl, topLevelMarker, project)
 	default:
 		log.Fatalf("%s: %s", src.Path, err)
-	}
-
-	if err != nil {
 		return nil
```

Comment:
`log.Fatalf` [will call](https://godoc.org/log#Fatalf) `os.Exit(1)` so we'll never get to this `return nil` line. We should remove that too.

Other than that -- nice catch, I think this was a bug before.

Reply:
Should I keep the Fatal log and remove return, or, should I replace it with Error log instead?

Reply:
Keep `log.Fatalf` and remove `return nil` I think. We don't propagate errors from this function so we should handle them (by terminating).

Reply:
Return removed.